haskell/package.yaml

```
@@ -0,0 +1,124 @@
+# SPDX-FileCopyrightText: 2021 Arthur Breitman
+# SPDX-License-Identifier: LicenseRef-MIT-Arthur-Breitman
+
+name:                segmented-cfmm-ligo-meta
+version:             0.0.1.0
+license:             MIT
+license-file:        LICENSE
+author:              Serokell, Arthur Breitman
+git:                 git@github.com:serokell/segmented-cfmm.git
+copyright:           2021 Arthur Breitman
+
+extra-source-files:
+- README.md
+
+description:         Tools and tests for the LIGO Segmented CFMM contract.
```

```suggestion
description:         Tools and tests for the Segmented CFMM contract.
```
No need to specify LIGO IMO (it's the only contract implementation that there is).

---

Remove all the 'LIGO' words